luigi/scheduler.py

```diff
@@ -1482,7 +1253,8 @@ def task_list(self, status='', upstream_status='', limit=True, search=None, max_
             pre_count = self._state.get_active_task_count_for_status(status)
             if limit and pre_count > count_limit:
                 return {'num_tasks': -1 if upstream_status else pre_count}
-        self.prune()
+
+        # self.prune()
```

Comment:
?

Reply:
Ah, holdover from when pruning was expensive (hit the store). Now that we've optimized this a bit, we can probably remove this line of change (although pruning here _is_ unnessecary)

Reply:
Actually, I think I'll just remove it unless you have objections -- we prune loads of other times, including every time a task's dep graph is looked at, so it'll still happen often -- and this is just called when someone loads the UI so it already isn't a consistent source of pruning, it'll just slow the UI down 😕